WalletWasabi.Gui/Controls/LockScreen/PinLockScreenViewModel.cs

```diff
@@ -65,19 +56,12 @@ public PinLockScreenViewModel(LockScreenViewModel lockScreenViewModel)
 			this.WhenAnyValue(x => x.PinInput)
 				.Throttle(TimeSpan.FromSeconds(1))
 				.ObserveOn(RxApp.MainThreadScheduler)
+				.Where(x => !string.IsNullOrWhiteSpace(x))
```

Comment:
Shouldn't it be after throttle? Checking a local variable doesn't need to be forced to the ui thread.

Reply:
Yes, fixed.